README.md

```diff
@@ -39,3 +39,24 @@ When we took on the task of importing the standards, a few principles guided our
 * **Easy to fix**: Converting 50 state standards is a monumental task (and one we're grateful ASN takes on). There will be typos and we want users to find them and fix them. Thus, each set of standards is versioned and changes can be submitted for approval.
 
 * **Easy to add to**: Schools & districts create their own standards (outside of their state). They should be able to publish those standards and share them with their vendors.
+
+## Development
+
+There are a few tasks that must be completed to get running with a local copy of the project for development purposes:
+
+* Sign up with [Algolia](https://www.algolia.com) and obtain your Application ID and Admin API Key.
+* Set the Algolia keys in your env: `export ALGOLIA_APPLICATION_ID=[APPID]; export  ALGOLIA_API_KEY=[KEYID]` (to make this persistent, set in your bash profile)
+* Install [MongoDB](https://www.mongodb.org/)
+* Run `mongorestore sources/jurisdiction-mongodb-dump` to do an initial import of jurisdictions
+* Clone the repo and run `bundle install` to get all the gems you will need
+* Run `bundle exec ruby importer/importer.rb` to import all the standards data into your local mongo (this requires internet access, and will take a while, go get a snack)
```

Comment:
Do you think this is better than having people doing a `mongorestore` of the whole DB? I hadn't considered other people would run the whole import script, but I like the idea. It means that there are other people who can maintain the import script and improve it. Also, we should make sure the upload to algolia section doesn't run by default so people doing this in dev mode aren't sending 450,000+ standards to algolia.

Also, the importer file needs to be broken up at some point. It's really two things:

1. Parse a JSON of the documents from ASN
2. Send each of the document URLs through the ETL process.

While breaking it up isn't causing a problem now, I think it makes it hard to immediately crock the dual purposes of the file. And the first goal (parse the doc from JSON) is part a flow that needs some help. Right now, I just download a list of standard documents from ASN and mung it (by hand) very slightly (as in, rename 'documents' to 'data' and remove the JSONP wrapper.) Ideally, that should all be automated.

Whenever we tackle the importer, we should break up the file.